### PR TITLE
Fixing metadata tests failures

### PR DESCRIPTION
**Pull request by prasa7**

### CODE REVIEW

File: modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java [hunk 1]
```diff
@@ -53,98 +54,117 @@
...
-                new RegistryProviderUtil().getWSRegistry(automationContext);
-
-        governance = new RegistryProviderUtil().getGovernanceRegistry(wsRegistry, automationContext);
+        String sessionCookie = new LoginLogoutClient(automationContext).login();
+
+        propertiesAdminServiceClient = new PropertiesAdminServiceClient(backendURL, sessionCookie);
+        resourceAdminServiceClient = new ResourceAdminServiceClient(backendURL, sessionCookie);
+        WSRegistryServiceClient wsRegistryServiceClient = new RegistryProviderUtil().getWSRegistry(automationContext);
+
+        governance = new RegistryProviderUtil().getGovernanceRegistry(wsRegistryServiceClient, automationContext);
```
Comment (prabathabey) [inline modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java]: May be it's good to change the variable name "governance" to something meaningful as the current name doesn't make much sense as to what it is used for.

File: modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java [hunk 2]
```diff
@@ -184,17 +211,9 @@ public void deleteResources()
         for (Service service : serviceManager.getAllServices()) {
             if (service.getQName().getLocalPart().equals("Info")) {
                 serviceManager.removeService(service.getId());
-            } else if (service.getQName().getLocalPart().equals("WeatherForecastService")) {
+            } else if (service.getQName().getLocalPart().equals("GeoIPService")) {
```
Comment (nuwand) [inline modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java]: The constant String literal should be the first part of the 'equals' check. 
Ex: else if("GeoIPService".equals(....)). 
Better to follow it as a practise, it avoids possible NPEs.

File: modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java (hunk 1 above)
Comment (prasa7) [inline modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java]: Hi,
Thanks for the comment. Fix is done in pull request[1] commit[2].
[1] - https://github.com/wso2/product-greg/pull/71
[2] - https://github.com/prasa7/product-greg/commit/b45f2f321b13765dc123a64ae55014c9eb21474c

File: modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java (hunk 2 above)
Comment (prasa7) [inline modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java]: Hi,
Thanks for the comment. Fix is done in pull request[1] commit[2].
[1] - https://github.com/wso2/product-greg/pull/71
[2] - https://github.com/prasa7/product-greg/commit/b45f2f321b13765dc123a64ae55014c9eb21474c

**Pull request merged**